### PR TITLE
gdome2: remove gdome2

### PR DESCRIPTION
**Pull request by ajgon**
As pointed out by @emilazy [in this PR](https://github.com/NixOS/nixpkgs/pull/369133#issuecomment-2564745574), gdome2 is old, unmaintained library which is not used by anything (was removed from workrave long time ago). This PR kills it (and moves `workrave` to new packages directories in a process).

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/wo/workrave/package.nix [hunk 1]
```diff
@@ -86,7 +82,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = with lib; {
-    broken = (stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64);
+    broken = stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64;
```
Comment (emilazy) [inline pkgs/by-name/wo/workrave/package.nix:85]: It builds fine on `aarch64-linux` now, so this can just be dropped (in a separate commit, please).

**Review by emilazy — CHANGES_REQUESTED** [review]
Thank you! Two necessary things and one suggestion:

* Please split the commits so that the `workrave` move to `by-name` is separate from the `gdome2` removal.

* Please add `gdome2 = throw "…"; # Added …` to `aliases.nix`  to explain the removal.

**Comment by emilazy** [comment]
Could you move the dropping of `gdome2` from `workrave` either into the commit that removes `gdome2` or its own commit so that the `by-name` commit is only doing what’s necessary for the move? Looks great to me otherwise!

**Comment by ajgon** [comment]
Done.

**Review by emilazy — APPROVED** [review]
Looks great, thank you!

**Pull request merged**